aqua/alacritty/Portfile

```diff
@@ -275,13 +266,19 @@ cargo.crates \
     walkdir                          2.3.1  777182bc735b6424e1a57516d35ed72cb8019d85c8c9bf536dccb3445c1a2f7d \
     wasi      0.9.0+wasi-snapshot-preview1  cccddf32554fecc6acb585f82a32a72e28b48f8c4c1883ddfeeeaa96f7d8e519 \
     wayland-client                  0.23.6  af1080ebe0efabcf12aef2132152f616038f2d7dcbbccf7b2d8c5270fe14bcda \
+    wayland-client                  0.27.0  ab702fefbcd6d6f67fb5816e3a89a3b5a42a94290abbc015311c9a30d1068ae4 \
     wayland-commons                 0.23.6  bb66b0d1a27c39bbce712b6372131c6e25149f03ffb0cd017cf8f7de8d66dbdb \
+    wayland-commons                 0.27.0  e972e9336ad5a9dd861b4e21ff35ad71d3e5c6b4803d65c39913612f851b95f1 \
+    wayland-cursor                  0.27.0  539f346e1a3f706f38c8ccbe1196001e2fb1c9b3e6b605c27d665db2f5b60d41 \
     wayland-protocols               0.23.6  6cc286643656742777d55dc8e70d144fa4699e426ca8e9d4ef454f4bf15ffcf9 \
+    wayland-protocols               0.27.0  f3d6fc54b17b98b5083bc21ae3a30e6d75cb4b01647360e4c3a04648bcf8781d \
     wayland-scanner                 0.23.6  93b02247366f395b9258054f964fe293ddd019c3237afba9be2ccbe9e1651c3d \
+    wayland-scanner                 0.27.0  030f56009d932bd9400bb472764fea8109be1b0fc482d9cd75496c943ac30328 \
     wayland-sys                     0.23.6  d94e89a86e6d6d7c7c9b19ebf48a03afaac4af6bc22ae570e9a24124b75358f4 \
+    wayland-sys                     0.27.0  8bdeffbbb474477dfa2acb45ac7479e5fe8f741c64ab032c5d11b94d07edc269 \
```

Comment:
I don't know how this cargo stuff works. Is it normal to need more than one version of a dependency?

Reply:
I'm not sure if it's normal, but this is generated entirely from the upstream project's  `Cargo.lock` file.

Here's where the duplicate `wayland-sys` packages are specified in this version's (`0.5.0`) `Cargo.lock`: https://github.com/alacritty/alacritty/blob/9454a369c37d5b033c4430764788e43a69e6017f/Cargo.lock#L2070